aio/gulp/conf.js

```diff
@@ -69,6 +69,11 @@ const version = {
  */
 const imageNameBase = 'kubernetes-dashboard';
 
+/**
+ * Arguments
+ */
+const argv = minimist(process.argv.slice(2));
```

Comment:
What does this exactly mean? Why there is `.slice(2)`?

Reply:
`process.argv` has [0] as `node`, [1] as `gulp` and [2] as `serve`. So we need after them.

Reply:
Makes sense, thank you.